apps/ui/src/networks/evm/constants.ts

```diff
@@ -18,14 +18,14 @@ import { MAX_SYMBOL_LENGTH } from '@/helpers/constants';
 
 // !NOTE: hardcode change
 export const MAX_MERKLE_VOTING_STRATEGIES = 100;
-export const MERKLE_VOTING_STRATEGY_ADDRESS = evmNetworks['skale-mainnet'].Strategies.MerkleVoting;
+export const MERKLE_VOTING_STRATEGY_ADDRESS = evmNetworks['europa-mainnet'].Strategies.MerkleVoting;
 export const COOLDOWN_PERIOD = 10; // * seconds
 export const MAX_ACTIVE_PROPOSALS = 100;
 // !NOTE: Update this after space deployed
-export const EXECUTION_STRATEGY_ADDRESS = '0xdF2Fb38249640C1D7ee6f0347970C4D3059f8D63';
-export const QUORUM = '10000000000000000000000000'; // * 10 million
-export const SPACE_ADDRESS = '0x579f0121f678Da51a66fFC0bf6059fF87f9045E5';
-export const REDIRECT_URL = `/skale-mainnet:${SPACE_ADDRESS}`;
+export const EXECUTION_STRATEGY_ADDRESS = '0x7659C8469c50218EfC881382A85D919aCa22b200';
+export const QUORUM = '100000000000000000000000'; // * 10 million
```

Comment:
@rajkharvar this is not 10M

Reply:
updated in [519e865](https://github.com/razor-network/sx-monorepo/pull/13/commits/519e865f9396ea8204585c5fdef1794b166d4d8a)